e2e/_suites/fleet/fleet.go

```diff
@@ -335,7 +333,7 @@ func (fts *FleetTestSuite) anAgentIsDeployedToFleetWithInstallerAndFleetServer(i
 	fts.CurrentTokenID = enrollmentKey.ID
 
 	var fleetConfig *kibana.FleetConfig
-	fleetConfig, err = deployAgentToFleet(agentInstaller, containerName, fts.CurrentToken)
+	fleetConfig, err = deployAgentToFleet(agentInstaller, fts.deployer, containerName, fts.CurrentToken)
```

Comment:
This is super NIT: as we commented online yesterday, the hierarchy tells us that the deployer has more importance than the installer. Wdyt about moving the deployer param to the first place in the method signature?

Reply:
Yea once I get the installer code in place, I think we'll be calling install commands from the deployer anyway and that agentInstaller param will go away